README.md

```diff
@@ -1,10 +1,10 @@
 # PSR-7 Storage-less HTTP Sessions
 
-[![Build Status](https://travis-ci.org/Ocramius/PSR7Session.svg)](https://travis-ci.org/Ocramius/PSR7Session)
-[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/Ocramius/PSR7Session/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/Ocramius/PSR7Session/?branch=master)
-[![Code Coverage](https://scrutinizer-ci.com/g/Ocramius/PSR7Session/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/Ocramius/PSR7Session/?branch=master)
-[![Packagist](https://img.shields.io/packagist/v/ocramius/psr7-session.svg)](https://packagist.org/packages/ocramius/psr7-session)
-[![Packagist](https://img.shields.io/packagist/vpre/ocramius/psr7-session.svg)](https://packagist.org/packages/ocramius/psr7-session)
+[![Build Status](https://travis-ci.org/psr7-sessions/storageless.svg)](https://travis-ci.org/psr7-sessions/storageless)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/psr7-sessions/storageless/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/psr7-sessions/storageless/?branch=master)
+[![Code Coverage](https://scrutinizer-ci.com/g/psr7-sessions/storageless/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/psr7-sessions/storageless/?branch=master)
+[![Packagist](https://img.shields.io/packagist/v/psr7-sessions/storageless.svg)](https://packagist.org/packages/psr7-sessions/storageless)
+[![Packagist](https://img.shields.io/packagist/vpre/psr7-sessions/storageless.svg)](https://packagist.org/packages/psr7-sessions/storageless)
 
 **PSR7Session** is a [PSR-7](http://www.php-fig.org/psr/psr-7/)
```

Comment:
Maybe mention that the storageless package is not only about session usage in PSR-7 based applications but also without storing anything (it sounds a bit stupid when put like this but is just to differentiate from the other possible packages of the org)?